<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcar</class>
 <widget class="QDialog" name="addcar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="pnglabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../photo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="addCarSectionLineEdit">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: grey;
border-radius: 10px;
border:5px solid black;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="addCarLabel">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;</string>
   </property>
   <property name="text">
    <string>Add Car </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addCarbackgroundLineEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>611</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:grey;
border-radius:8px;
border:5px solid black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="carNoLabel">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Car No  :</string>
   </property>
  </widget>
  <widget class="QLabel" name="brandLabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>280</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Brand  : </string>
   </property>
  </widget>
  <widget class="QLabel" name="carTypeLabel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>370</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Car Type : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>610</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;
border-radius:10px;
border:3px solid black;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCarButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>610</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:green;
border-radius:12px;
border:3px solid black;</string>
   </property>
   <property name="text">
    <string>Add Car</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="carNoLineEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>190</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="fareLineEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>500</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="fareLabel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>500</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Fare : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brandLineEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>280</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="carTypeListWidget">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>370</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>610</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:grey;
border-radius:12px;
border:3px solid black;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
